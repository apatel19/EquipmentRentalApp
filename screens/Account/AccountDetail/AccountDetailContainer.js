import React from 'react';
import {View, Text, StyleSheet} from 'react-native';

const AccountDetailContainer = props => {
  return (
    <View>
      <Text>Account Detail</Text>
    </View>
  );
};

const styles = StyleSheet.create({});

export default AccountDetailContainer;
