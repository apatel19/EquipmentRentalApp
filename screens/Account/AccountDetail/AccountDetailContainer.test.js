import React from 'react';
import AccountDetailContainer from './AccountDetailContainer';
import renderer from 'react-test-renderer';

test('Card snapshot match test', () => {
  const tree = renderer.create(<AccountDetailContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
