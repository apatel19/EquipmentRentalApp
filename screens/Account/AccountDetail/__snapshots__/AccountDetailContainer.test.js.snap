// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot match test 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "margin": 20,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            style={
              Object {
                "fontSize": 16,
                "marginVertical": 8,
              }
            }
          >
            Name
          </Text>
          <TextInput
            allowFontScaling={true}
            rejectResponderTermination={true}
            style={
              Object {
                "borderBottomColor": "#ccc",
                "borderBottomWidth": 1,
                "borderColor": "black",
                "borderWidth": 1,
                "paddingHorizontal": 2,
                "paddingVertical": 5,
              }
            }
            title="Change Name"
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          style={Object {}}
        >
          <Text
            style={
              Object {
                "fontSize": 16,
                "marginVertical": 8,
              }
            }
          >
            Email
          </Text>
          <TextInput
            allowFontScaling={true}
            rejectResponderTermination={true}
            style={
              Object {
                "borderBottomColor": "#ccc",
                "borderBottomWidth": 1,
                "borderColor": "black",
                "borderWidth": 1,
                "paddingHorizontal": 2,
                "paddingVertical": 5,
              }
            }
            title="Change Email"
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          style={Object {}}
        >
          <Text
            style={
              Object {
                "fontSize": 16,
                "marginVertical": 8,
              }
            }
          >
            Password
          </Text>
          <TextInput
            allowFontScaling={true}
            rejectResponderTermination={true}
            style={
              Object {
                "borderBottomColor": "#ccc",
                "borderBottomWidth": 1,
                "borderColor": "black",
                "borderWidth": 1,
                "paddingHorizontal": 2,
                "paddingVertical": 5,
              }
            }
            title="Change Password"
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          style={Object {}}
        >
          <View
            accessibilityRole="button"
            accessibilityStates={Array []}
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {},
                ]
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#007AFF",
                      "fontSize": 18,
                      "padding": 8,
                      "textAlign": "center",
                    },
                  ]
                }
              >
                Save
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
